server/controllers/country.controller.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+const responseUtil = require('../utils/responseUtils');
+const MessageUtil = require('../utils/messageUtil');
+const countryService = require("../services/country.service");
+const { v4: uuidv4 } = require('uuid');
+
+exports.getAllCountry = async (req, res) => {
+    try {
+        const response = await countryService.findAll();
+        if (response) {
+            responseUtil.successResponse(res, MessageUtil.success, response);
+        } else {
+            responseUtil.failResponse(res, MessageUtil.requestedDataNotFound, response);
```

Comment:
This is also a great option of explicitly specifying the reason for error response, rather than throwing errors.